worlds/ladx/LADXR/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License

```

Comment:
It should be clear this license is for LADXR or just fall under the MIT of the AP project

Reply:
addressed